pkg/kev/converter/kubernetes/transform.go

```diff
@@ -1660,20 +1660,16 @@ func (k *Kubernetes) updateKubernetesObjects(projectService ProjectService, obje
 	}
 
 	// @step add PVCs to objects
-	if pvcs != nil {
```

Comment:
What changed to allow removing this check?

Reply:
nil is the zero value for slices so for range works on them it just doesn't iterate. No need to protect with a nil check

Reply:
https://play.golang.org/p/dB_0-Xoq6HY